hack/e2e-test.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
I think the `local-e2e` target was supposed to serve this purpose - can we merge the two if the existing target does not work?

Reply:
This script utilizes the `local-e2e` target, but you are right it is probably preferable to have that target call this script for setup instead. I was thinking of this as more of a _helper_ that wasn't required to run the e2e tests locally, but just did some setup and extra logging. Do you think that this would be useful every time, and thus should be merged with the `local-e2e` target?